user-operator/scripts/user_operator_run.sh

```diff
@@ -14,4 +16,4 @@ fi
 
 export JAVA_CLASSPATH=lib/io.strimzi.@project.build.finalName@.@project.packaging@:@project.dist.classpath@
 export JAVA_MAIN=io.strimzi.operator.user.Main
-exec ${STRIMZI_HOME}/bin/launch_java.sh
+exec "${STRIMZI_HOME}"/bin/launch_java.sh
```

Comment:
Normally you would quote the whole string I.e. exec "${STRIMZI_HOME}/bin/launch_java.sh"

I think in practice is has little difference but it makes it a little clearer to read